src/custom/utils/analytics/transactionEvents.ts

```diff
@@ -0,0 +1,79 @@
+import { Category, _reportEvent } from './index'
+
+type ExpirationAction = 'Default' | 'Custom'
+export function orderExpirationTimeAnalytics(action: ExpirationAction, value: number) {
+  _reportEvent({
+    category: Category.ORDER_EXPIRATION_TIME,
+    action: `Set ${action} Expiration Time`,
+    value,
+  })
+}
+
+type wrapTransactionType = 'Send' | 'Sign' | 'Reject' | 'Error'
```

Comment:
why lowecase type?
Also, here is not a `Type` but is an `Action`

Reply:
U were not even consistent with `ClaimAction`

Reply:
This one slipped through, I forgot to updated it. Good catch